processor/elasticprocessor/internal/hostmetrics/process.go

```diff
@@ -129,8 +129,10 @@ func addProcessMetrics(metrics pmetric.MetricSlice, dataset string) error {
 	cpuTimeValue = total * 1000
 	systemCpuTime = systemCpuTime * 1000
 	userCpuTime = userCpuTime * 1000
+	timeDiff = timestamp.AsTime().UnixMilli() - startTime
```

Comment:
i was confused for a second about what this value is - can we give it a more descriptive name?

perhaps `processRuntime` or something?